src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private final String name;

```

Comment:
Do you need Fruit model ? u can use String fruit inside FruitTransaction and parse all input file with this model